formbricks/formbricks
# Validate for E-Mail in User Verification

## What does this PR do?
Uses the industry standard `validator` library and validates for email param passed in the **Email Verification** modal!

## Type of change

## How should this be tested?
Enter any other value in the URL apart from an email and it should not work!

However, for the most robust approach: we should think of not displaying this user submitted field directly and rather have something simple as: `We sent an email to your email ID, Please click the link in that email to activate your account.` This will cause a UX downside though! Let me know if this is something that we are open for!

## Checklist
